about.html

```diff
@@ -11,63 +11,110 @@
 
 </head>
 
+<div class="navContainer">
+  <img class="lambdaImage">
+  <img src="./img/lambda-black.png">
+  </img>
+  
+  
+  <nav class="topButtons">
+     <a href="./index.html"> <div class="button1">Home</div> </a>
+      <a href="./about.html"><div class="button2">About</div> </a>
+     <a href="#"> <div class="button3">Products</div></a>
+      <div class="button4">Blog</div>
+      <div class="button5">Contact</div>
+  </nav>
+  </div>
+
+  <div class="headerImage">
+    <img src="./img/jumbo-about.png">
+    </div>
+
 <body>
-    <div class="container about-page">
+<div class="container about-page">
      
```

Comment:
The about.html page is a good place to use semantic HTML to layout your page so it is more descriptive what each area of the page is for.